### ISSUE TITLE
Add more information to commit message

### ISSUE THREAD

**Issue by hds** [MarcoIeni/release-plz#1355]
## Motivations

The PR which is created for a release has lots of useful information in it. The previous version, the version to be released, changelog, git history covering the changes, and more.

For reference, here's an example from a test project.

<img width="912" alt="hds_test0-v0 0 4-release-pr" src="https://github.com/MarcoIeni/release-plz/assets/89589/33ce9c42-a673-4cf6-a19a-b4d5cba9161a">

However, the commit message ends up looking like:

```COMMIT_EDITMSG
chore: release (#6)

Signed-off-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
```

It's missing all of that useful information.

- Would you like to implement this feature? maybe 😅

## Solution

I would like to see the possibility of having all the same information that goes into a PR in the commit message. It's a more durable information store than GitHub (or alternative platforms).

It would probably be necessary to format it differently. While markdown in a commit message is often not too hard to read, adding HTML tags can make it more difficult.

An example which does what I want, but needs some more polish is what we've implemented for tokio-console:
https://github.com/tokio-rs/console/commit/05484b136f8400dd2b91ee8155e09954a9da448c

It's also using git-cliff for release notes, which might be where some more work is needed.

## Alternatives

The user can copy and paste the PR description into the commit message when submitting, but this isn't an ideal workflow in my opinion.

**Comment by MarcoIeni** [comment]
Hey Hayden 👋
Would this solve your issue? https://github.blog/changelog/2022-08-23-new-options-for-controlling-the-default-commit-message-when-merging-a-pull-request/

Tldr from my understanding in github you can set the commit message as the the pr description by default

**Comment by hds** [comment]
@MarcoIeni Sorry! I completely missed the movement on this issue.

I think that this is a good general solution to my concern. In some cases, using this setting isn't desired because a team/project's workflow revolves around force pushing and keeping the commit message up to date, rather than the PR message.

But closing this issue is the right way to go - I think I might be a bit special here.

Thank you for your help!

**Comment by MarcoIeni** [comment]
>  I think I might be a bit special here.

I don't think so, you raised a valid concern.

Hopefully this solution is good enough for you. If it's not, feel free to open another issue or re-open this one :)
